portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -54,19 +58,23 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     PreparedQuery results = datastore.prepare(query);
 
     comments = new ArrayList<>();
+    emails = new ArrayList<>();
     for (Entity entity : results.asIterable()) {
       if (commentLimitChoice == 0) {
         break;
       }
       String commentEntry = (String) entity.getProperty("comment-text");
+      String complimentEmail = (String) entity.getProperty("email");
```

Comment:
I'm not sure what you are referring to with "compliment"?

Reply:
It's the email that is associated with the comment, but I can just rename it to "email" to keep things minimal(?)

Reply:
yea, I think email is better as compliment doesn't add much here.

Maybe if you had a prefix for all elements such as

entryComment
entryEmail 

but then entry doesn't add much there anyway.